barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -701,7 +701,8 @@ template <typename Builder = UltraCircuitBuilder> void gateCount(const std::stri
     for (auto constraint_system : constraint_systems) {
         auto builder = acir_format::create_circuit<Builder>(
             constraint_system, 0, {}, honk_recursion, std::make_shared<bb::ECCOpQueue>(), true);
-        auto circuit_size = builder.get_total_circuit_size();
+        builder.finalize_circuit();
```

Comment:
Shouldn't this also include the ensure_nonzero gates as well?

Reply:
that happens in finalisation

Reply:
also, does this fix the regression acir test?